snap/local/bin/source-env-file.sh

```diff
@@ -17,7 +17,10 @@ logger "edgex service override: : SERVICE_ENV=$SERVICE_ENV"
 
 if [ -f "$SERVICE_ENV" ]; then
     logger "edgex service override: : sourcing $SERVICE_ENV"
-    source "$SERVICE_ENV"
+    set -o allexport
+    source "$SERVICE_ENV" set
```

Comment:
Please update to:
- add the app name as syslog tag
- change SERVICE_ENV to something more clear